datanode/gateway/graphql/schema.graphql

```diff
@@ -2383,7 +2379,7 @@ enum WithdrawalStatus {
 type Deposit {
   "The Vega internal ID of the deposit"
   id: ID!
-  "The PartyID initiating the deposit"
+  "The initiating the deposit"
```

Comment:
'The party initiating the deposit' ?

Reply:
Addresssed here https://github.com/vegaprotocol/vega/pull/5895/commits/793f57749039ae2385cf429e737f2973ec75a55d#diff-be016f833f94c6069e96beb4873414c55bdfb7af1924c9c1b90eb709304ff90eR2382
Thanks!